server/views/components/list-header/list-header.njk

```diff
@@ -1,23 +1,11 @@
-<div class="row bg-cream  {{ {s:3, m:5, l:5} | spacingClasses({padding: ['top']}) }} }}">
+<div class="row bg-cream  {{ {s:3, m:5, l:5} | spacingClasses({padding: ['top', 'bottom']}) }} }}">
   <div class="container">
-    <div class="grid">
-      <div class="grid__cell">
-        <h1 class="{{ {s:'WB6', m:'WB5', l:'WB4'} | fontClasses }} {{ {s:0, m:0, l:0} | spacingClasses({margin: ['top']}) }} {{ {s:0, m:0, l:0} | spacingClasses({margin: ['bottom']}) }}">{{ model.name }}</h1>
-      </div>
-    </div>
+    <h1 class="{{ {s:'WB6', m:'WB5', l:'WB4'} | fontClasses }} no-margin">{{ model.name }}</h1>
 
     {% if model.description %}
-      <div class="grid">
-        <div class="{{ {s: 12, m: 12, l: 8, xl: 8} | gridClasses }}">
-          <p class="{{ {s:2, m:2, l:2} | spacingClasses({margin: ['top']}) }} {{ {s:0, m:0, l:0} | spacingClasses({margin: ['bottom']}) }}">{{ model.description }}</p>
-        </div>
+      <div class="{{ {s:2} | spacingClasses({margin: ['top']}) }}">
```

Comment:
I think the grid classes here should stay in – they prevented the description from becoming too wide at larger screen sizes.

Removing them above is probably OK since currently we only have one or two word headings.

Reply:
Nice shout, series would be a good example of this.

Reply:
Done